packages/e2e.sandbox.todoapp/src/useTodos.tsx

```diff
@@ -173,31 +173,30 @@ export function useSubscribe(mutate: KeyedMutator<SimpleTodo[]>) {
         "isComplete",
         "title",
       ], {
-        onChange(objectUpdates) {
+        onChange(objectUpdate) {
           // index incoming objects by apiName and then by pk value
           const byApiNameByPK = new Map<
             string,
             Map<
-              (typeof objectUpdates)[0]["object"]["$primaryKey"],
-              (typeof objectUpdates)[0]
+              (typeof objectUpdate)["object"]["$primaryKey"],
+              (typeof objectUpdate)
             >
           >();
-          for (const objectUpdate of objectUpdates) {
-            const byPk = byApiNameByPK.get(objectUpdate.object.$apiName);
-            if (byPk) {
-              byPk.set(objectUpdate.object.$primaryKey, objectUpdate);
-            } else {
-              byApiNameByPK.set(
-                objectUpdate.object.$apiName,
-                new Map([[objectUpdate.object.$primaryKey, objectUpdate]]),
-              );
-            }
+
+          const byPk = byApiNameByPK.get(objectUpdate.object.$apiName);
+          if (byPk) {
+            byPk.set(objectUpdate.object.$primaryKey, objectUpdate);
```

Comment:
what else is on `objectUpdate` ? Do we ant the api to be `.object` ?

Reply:
objectUpdate comes back with the state("was the object added/updated or deleted")